src/doc/en/developer/doctesting.rst

```diff
@@ -716,7 +716,7 @@ Use the ``--long`` flag to run doctests that have been marked with the
 comment ``# long time``. These tests are normally skipped in order to
 reduce the time spent running tests::
 
-    [roed@sage sage-6.0]$ sage -t src/sage/rings/tests.py
+    [roed@sage sage-6.0]$ ./sage -t src/sage/rings/tests.py
     Running doctests with ID 2012-06-21-16-00-13-40835825.
```

Comment:
This looks ugly. For general case, we can assume `sage` just runs the sage in the sage root directory. No?

Reply:
Most people don't have `.` in their `PATH`; given that the shown command line is specific about the directory from which it is run, I find it misleading to just have `sage` there

Reply:
I guess that most developers set up path so that `sage` just runs the right (the unique) sage.

We countlessly run "sage -t". I doubt a developer types "./sage -t" every time. 

If the developer has several sage installs so that missing "./" makes any difference, then I guess he knows well enough to be "misled".

But If you disagree and prefer to be safe, then I would not insist.

Reply:
> I doubt a developer types "./sage -t" every time.

Hm... I do that all the time

Reply:
:-) OK then.

Reply:
On the other hand, I am not happy with all different prompts appearing in the developer manual:

- `[roed@sage sage-6.0]$ ./sage ...`
- `[mkoeppe@sage sage]$ ./sage ...`
- `[mkoeppe@sage sage]$ sage ...`
- `[jdemeyer@sage sage-6.0]$ ./sage ...`
- `[jdemeyer@sage sage-6.0]$ sage ...`
- `root@39d693b2a75d:/sage# ./sage ...`
- `[alice@localhost sage]$ ./sage ...`

all of which could be simplified to one form. But definitely not to do in this PR.

Reply:
+1 on replacing all `@sage` with `@localhost`
and on replacing all versioned forms of the directory like `sage-6.0` by `sage`.

The examples that look like `    [mvngu@sage build]$ sage-6.0/sage -t my_sage_script.sage` are confused and should be changed too.

The `root@...` prompt is the actual form of the prompt in the container, shouldn't be changed.

Reply:
I've made this change in cec2873e0a